src/core/mount.c

```diff
@@ -1997,6 +1997,7 @@ static void mount_enumerate(Manager *m) {
         mnt_init_debug(0);
 
         if (!m->mount_monitor) {
+                uint64_t mount_rate_limit_interval = 1;
```

Comment:
we generally want all time values to be stored in µs internally, hence the rhs should be `1 * USEC_PER_SEC` here.

(note that parse_sec() will return µs too, hence no further manual multiplication necessary)